### DISCUSSION TITLE
opening date for media list twice double entries

### DISCUSSION THREAD

**Discussion by riemsesy** [sircharlo/meeting-media-manager#369] [General]
why do lines appear twice when I go back to the place where I create the medalist?
![2022-05-22 14_07_15-Window](https://user-images.githubusercontent.com/39587825/169694497-6dc83cdd-9bdc-486c-bcbc-86db500d4332.png)

**Comment by riemsesy** [comment]
and if I may add.. I can't find it in the translation file
Song  = Lied in NL
Paragraph = Alinea in NL

**Reply by sircharlo** [reply]
These strings are hard coded in English for now. I'll see if I can add them to the list of things that can be translated eventually.

**Comment by riemsesy** [comment]
and if I may add.. The blue lines showing what is played.. is the new? Very nice.

**Reply by sircharlo** [reply]
Yes, new with latest version. Just to help keep track of what was played or not. Glad you enjoy it!

**Comment by sircharlo** [comment]
In short: The “extra” lines correspond to the auto-generated MP4 files.

Explanation: 
M³ sees that a picture is available from a publication and will be added to the media for that day. (the media type icon shows that the media is a picture file)
The checkbox can be unchecked if you wish to skip that picture file.
M³ also shows you that a file already exists in the folder, but this is not a picture file but a video file (notice again the media icon).
The automatically generated MP4 files get overwritten as necessary.

By the way, you should not need to use MP4 conversion during hybrid meetings if using the media presentation mode in M³. MP4 conversion is only useful when sharing all media files through Zoom MP4 sharing, for remote meetings.

**Reply by riemsesy** [reply]
thanks for the tip about the mp4 conversion. before we used that to play pictures with vlc, but I can disable that. something learned.
